src/test/scala/it/pagopa/interop/agreementprocess/AgreementSuspensionSpec.scala

```diff
@@ -36,7 +36,8 @@ class AgreementSuspensionSpec extends AnyWordSpecLike with SpecHelper with Scala
         suspendedByConsumer = Some(true),
         suspendedByProducer = None,
         suspendedByPlatform = Some(false),
-        stamps = SpecData.suspensionByConsumerStamps
+        stamps = SpecData.suspensionByConsumerStamps,
+        suspendedAt = Some(SpecData.when)
```

Comment:
I know that hard coded values are actually the same, but to clear in the test what the logic is applied here, I suggest to set it like
```suggestion
        suspendedAt = SpecData.suspensionByConsumerStamps.suspensionByConsumer
```
same for other test cases